dhis-2/dhis-services/dhis-service-program-rule/src/main/java/org/hisp/dhis/programrule/engine/NotificationRuleActionImplementer.java

```diff
@@ -70,7 +69,6 @@ protected ExternalNotificationLogEntry createLogEntry(String key, String templat
     return entry;
   }
 
-  @Transactional(readOnly = true)
```

Comment:
why do we remove the transactional annotation?

Reply:
Because it was useless here.
This method is just calling a service that is already annotated.